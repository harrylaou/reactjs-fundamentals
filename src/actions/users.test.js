import {receiveUsers, RECEIVE_USERS} from './users'

test(`Return an Action tyoe ${RECEIVE_USERS}`, () => {
  const users = ["Francis"]
  expect(receiveUsers(users)).toEqual({
    type: RECEIVE_USERS,
    users
  })

  expect(receiveUsers(users)).toMatchSnapshot()
})