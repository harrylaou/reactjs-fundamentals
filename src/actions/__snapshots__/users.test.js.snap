// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Return an Action tyoe RECEIVE_USERS 1`] = `
Object {
  "type": "RECEIVE_USERS",
  "users": Array [
    "Francis",
  ],
}
`;
